doc/index.md

```diff
@@ -10,7 +10,7 @@ width: 125
 
 # Overview
 
-<h2 style="margin-top: 0.3em;">The powerful data exploration and application framework for Python</h2>
+<h1 style="margin-top: 0.3em;">The powerful data exploration and application framework for Python</h1>
 
```

Comment:
An alternative title (easier to read?) could be

"The powerful framework for data exploration and application development in Python"